pipelines/models/tensorflow-housing/README.md

```diff
@@ -0,0 +1,27 @@
+# How was this model generated?
+
+## Data source
+
+The model uses
+https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing,
+see
+https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html
+for more information about the data.
+
+## Generating model
+
+Command similar to the following was used to generate the model:
```

Comment:
```suggestion
A command similar to the following was used to generate the model:
```

Reply:
Thanks, updated in 8d26a2d4503e905be17c7548045bca67da6a905e.